### PR TITLE
Migrate multi-session-applications from Sanity to MDX

### PR DESCRIPTION
**Pull request by alexisintech**
/docs/custom-flows/multi-session-applications

This PR

- migrates this page from Sanity to MDX
- adds links to references
- adds image and optimized using Squoosh

Acknowledging that these pages probably need work, and need their code examples updated - but for the scope of "Sanity to MDX Migration", this can be revisited at a later time, with better resources.

For reference, this is what the page looked like before this PR:

https://github.com/clerkinc/clerk-docs/assets/98043211/65b5970f-6868-4f6e-8fb7-d66270925c2d

### CODE REVIEW

**Comment by BrandonRomano** [comment]
@alexisintech, one small ergonomic thing I recommend you can include for reviewers on these PRs:

Preview Links:
- [🔍 Preview of updates](https://docs-preview-319.clerkpreview.com/docs/custom-flows/multi-session-applications)
- [🔍 Old page](https://clerk.com/docs/custom-flows/multi-session-applications)

This will make it slightly quicker to review these PRs which I think will make it ever so slightly easier to get timely PRs!

**Review by BrandonRomano — CHANGES_REQUESTED** [review]
CleanShot 2023-09-21 at 12 33 02@2x

An image is not loading for me on the page.  Tried refreshing a few times to no avail.

**Comment by alexisintech** [comment]
> CleanShot 2023-09-21 at 12 33 02@2x
> 
> An image is not loading for me on the page. Tried refreshing a few times to no avail.

ahh, because the image was added in other pr!! that pr is now merged, so it should now be ok

File: docs/custom-flows/multi-session-applications.mdx [hunk 1]
```diff
@@ -1,5 +1,183 @@
...
 
-# This is a stub
+# Multi-session applications
+
+One of the most powerful features that Clerk provides out of the box is multi-session applications.
+
+A multi-session application is an application that allows multiple accounts to be signed-in from the same browser at the same time. The user can switch from one account to another seamlessly. Each account is independent from the rest and has access to different resources. It's most clearly explained by the default [`<UserButton />`][user-button-ref] below.
+
+<Images
+  width={880}
```
Comment (jescalan) [inline docs/custom-flows/multi-session-applications.mdx:13]: Image load is now fixed but it looked way too wide to me - this felt a bit more reasonable

```suggestion
  width={400}
```

**Review by jescalan — COMMENTED** [review]
Just one small image width thing

**Review by BrandonRomano — APPROVED** [review]
LGTM

**Pull request merged**